models/stg_greenhouse__job_opening.sql

```diff
@@ -23,13 +23,13 @@ final as (
     
     select 
         _fivetran_synced,
-        application_id,
-        close_reason_id,
+        cast(application_id as {{ dbt.type_string() }}) as application_id,
+        cast(close_reason_id as {{ dbt.type_string() }}) as close_reason_id,
         cast(closed_at as {{ dbt.type_timestamp() }}) as closed_at,
-        id as job_openining_id,
-        job_id,
+        cast(id as {{ dbt.type_string() }})as job_openining_id,
```

Comment:
why was `job_openining_id` originally spelled wrong lol

I guess we should leave it to avoid making field changes too.